app/models/git_hub_user.rb

```diff
@@ -15,8 +15,12 @@ def login
   # Public
   #
   def admin_organization_memberships
+    no_cache_headers = { 'Cache-Control' => 'no-cache, no-store' }
```

Comment:
@mastahyeti this is what the log looks like on `master` with a clear cache:

```ruby
>> github_user.admin_organization_memberships
HTTP Cache: [GET /user/memberships/orgs?per_page=100&state=active] miss, store
```

This is what it looks like hitting it a second time.
```ruby
>> github_user.admin_organization_memberships
HTTP Cache: [GET /user/memberships/orgs?per_page=100&state=active] fresh
```

This is what it looks like with the new changes
```ruby
>> github_user.admin_organization_memberships
HTTP Cache: [GET /user/memberships/orgs?per_page=100&state=active] unacceptable
```

According to https://github.com/plataformatec/faraday-http-cache#instrumentation `:unacceptable` means that the request did not go through the cache at all.

Does this look right?